### PR TITLE
[17.0] [MIG] rma_account 

### PR DESCRIPTION
**Pull request by agyamuta**

### CODE REVIEW

File: rma_account/__manifest__.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 {
     "name": "RMA Account",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "license": "LGPL-3",
     "category": "RMA",
     "summary": "Integrates RMA with Invoice Processing",
     "author": "ForgeFlow",
-    "website": "https://github.com/ForgeFlow/stock-rma",
+    "website": "https://github.com/OCA/",
```
Comment (CarlosVForgeFlow) [inline rma_account/__manifest__.py:11]: Why this change of the website? This module is still located in the https://github.com/ForgeFlow/stock-rma repo.

**Comment by AaronHForgeFlow** [comment]
When working with rma groups, I cannot see the wizard to create rma lines from an invoice. I see in the code the wizard is there but I cannot see a way to open it. I remember that that was an option in previous versions of the module.

**Comment by AaronHForgeFlow** [comment]
Hi @agyamuta sorry I had to fix the history in the base 17.0 branch, as it was completely incorrect. That action automatically closed your pull request as the history was unrelated.

 I create this PR for the supersede: https://github.com/ForgeFlow/stock-rma/pull/513

I respected the code in this PR and the commit authorship
